src/Components/v2/__tests__/Carousel.test.tsx

```diff
@@ -46,7 +46,10 @@ describe("Carousel", () => {
           }}
         />
       </MockBoot>
-    )
+    ).renderUntil(n => {
+      return n.html().search("is-selected") > 0
```

Comment:
```suggestion
      return n.html().includes("is-selected")
```

It's a minor thing but `includes` has a boolean api.